cluster/saltbase/salt/fluentd-es/fluentd-es.manifest

```diff
@@ -2,7 +2,10 @@ version: v1beta2
 id: fluentd-to-elasticsearch
 containers:
   - name: fluentd-es
-    image: kubernetes/fluentd-elasticsearch:1.1
+    image: kubernetes/fluentd-elasticsearch:1.2
+    env:
+      - name: "FLUENTD_ARGS"
+        value: "-qq"
```

Comment:
Probably not, but I am following the pattern used in Grafana/Heapster. Quotes removed.